src/access.c

```diff
@@ -736,7 +746,7 @@ access_get(struct sockaddr_storage *src, const char *username, verify_callback_t
                        superuser_username, superuser_password))
       return access_full(a);
   } else {
-    s = alloca(50);
+    char s[50];
```

Comment:
doesn't seem to be related to this patchset? though (i know below its hardcoded as well) should really be a define/sizeof combo :)